lib/rubocop/cop/rails/logger_block.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks that blocks are passed to Rails.logger instead of values.
+      #
+      # @example
+      #   #bad
+      #   Rails.logger.debug 'Beginning import.'
```

Comment:
What https://guides.rubyonrails.org/debugging_rails_applications.html#impact-of-logs-on-performance is talking about is the case of string interpolation. Maybe non interpolation string can be allowed?